bin/run-tests

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+for D in exercises/*; do
+    CURRENT_DIR=$(pwd)
+
+    if [ -d "${D}" ]; then
+        # Get the exercise name from the test file
+        TEST_FILE=$(basename $(ls ${D}/test/test*))
+        STRIPPED_OF_EXTENSION="${TEST_FILE%.*}"
+        EXERCISE_NAME="${STRIPPED_OF_EXTENSION:5}"
+
+        # Copy the examples with the correct name for the exercise
+        if [ -e "${D}/src/example.c" ]
+        then
+          cp -n ${D}/src/example.c ${D}/src/${EXERCISE_NAME}.c
+        fi
+
+        if [ -e "${D}/src/example.h" ]
+        then
+          cp -n ${D}/src/example.h ${D}/src/${EXERCISE_NAME}.h
+        fi
+
+        # Make it!
+        { cd ${D};
+          echo "Running tests for ${EXERCISE_NAME}";
+          make clean >> /dev/null; # I'm just trying to suppress the output (I'm sure there's a much better way)
```

Comment:
Directing output to /dev/null works fine. This line shouldn't be necessary on Travis but it certainly doesn't hurt and makes using the script locally easier, so I say leave it in.

Maybe remove the comment though.